src/widget/text-print/text-print.js

```diff
@@ -0,0 +1,48 @@
+import Widget from '../../js/widget';
+
+/**
+ * Clone text input fields value into new print-only element.
+ * This is an unusual way to implement a feature, because it is not an actual widget,
+ * but this is the easiest way to do it.
+ */
+class TextPrintWidget extends Widget {
+    /**
+     * @type string
+     */
+    static get selector() {
+        return '.question:not(.or-appearance-autocomplete) > input[type=text]:not(.ignore)';
+    }
+
+    _init() {
+        this.question.classList.add( 'or-text-print-initialized' );
+
+        const className = 'print-input-text';
+        const printElement = document.createElement( 'div' );
+        printElement.classList.add( className );
+        printElement.classList.add( 'print-only' );
+
+        if ( getComputedStyle( this.element ).order !== '' ) {
+            printElement.style.order = parseInt( getComputedStyle( this.element ).order, 10 ) + 1;
+        }
+        this.element.after( printElement );
+
+        this.widget = this.element.parentElement.querySelector( `.${className}` );
+        this._copyValue();
+
+        this.element.addEventListener( 'input', () => {
+            this._copyValue();
+        } );
+    }
+
+    _copyValue() {
+        if (this.element.value !== '') {
```

Comment:
Oops. Sorry I overlooked this one before! The if() statement keeps a ghost of an old value in the print element if the value in the input is cleared!

See http://localhost:8005/?xform=https://github.com/enketo/enketo-core/files/3852678/print-long-text.xml.txt

1. clear value in first question
2. print 

first two questions on printout show old value but should be empty.

Reply:
Ah, I missed that. Thanks!